src/System.Text.Json/src/System/Text/Json/Serialization/JsonExtensionDataAttribute.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Text.Json.Serialization
+{
+    /// <summary>
+    /// When placed on a property of type <see cref="System.Collections.Generic.Dictionary{TKey, TValue}"/>, any
+    /// properties that do not have a matching member are added during deserialization and written during serialization.
+    /// The TKey value must be <see cref="string"/> and TValue must be <see cref="JsonElement"/> or <see cref="object"/>.
```

Comment:
Also is there a to way to add the string\Dictionary\object values into the xml doc directly instead of TKey and TValue? If so I could avoid this comment.

Reply:
@mairaw, @ rpetrusha - can you help answer @steveharter's question?

Reply:
Not that I'm aware unless you want to style as code instead of an API link.